doc/howto/dev/contribute_to_paddle_en.md

```diff
@@ -1,219 +0,0 @@
-# Contribute Code
-
-We sincerely appreciate your contributions. You can use fork and pull request
-workflow to merge your code.
-
-## Code Requirements
-- Your code comments must be fully documented by
-  [Doxygen](http://www.stack.nl/~dimitri/doxygen/) style.
-- Make sure the compiler option `WITH_STYLE_CHECK` is on and the compiler
-  passes the code style check.
-- All code must have unit test.
-- Pass all unit tests.
```

Comment:
Can we remain these Code Requirements?

Reply:
Good idea. I moving it after the *Workflow* section, as the *Code Standard* section.